### PR TITLE
fix: menu behavior when inside modal

### PR DESCRIPTION
**Pull request by nikolaykost**
##### Checklist

Before taking this PR from the draft, please, make sure you have done the following:

- [ ] Pipeline is passed
- [ ] Tests are added (including unit tests and stories in the storybook)
- [ ] Tests are passed successfully
- [ ] If you're adding a new component/new props, add stories that describe how this component/prop works
- [ ] Changeset(s) is(are) added
- [ ] You have passed the threshold of the library size
- [ ] Commit message follows [commit guidelines](https://github.com/cube-js/cube-ui-kit/blob/main/CONTRIBUTING.md)

Closes:  N/A 

#

### Other information

When using MenuTrigger inside DialogContainer or any Popover it doesn't work properly.

### CODE REVIEW

File: .changeset/weak-dolls-move.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@cube-dev/ui-kit": patch
+---
+
+fix(menu): menu trigger behavior when inside modal
```
Comment (MrFlashAccount) [inline .changeset/weak-dolls-move.md:5]: Please, don't write changelog as you write your commits, prefer a human-oriented style instead, e.g. `Fixed bug when menu doesn't open within a modal`

File: src/components/pickers/Menu/Menu.stories.tsx [hunk 2]
```diff
@@ -88,6 +90,35 @@ export const Default = ({ ...props }) => {
   );
 };
 
+export const InsideModal = () => {
```
Comment (MrFlashAccount) [inline src/components/pickers/Menu/Menu.stories.tsx:93]: I think we need a play function for the story to make sure everything works

**Review by MrFlashAccount — CHANGES_REQUESTED** [review]

File: src/components/pickers/Menu/Menu.stories.tsx (hunk 2 above)
Comment (JiLiZART) [reply src/components/pickers/Menu/Menu.stories.tsx:93]: Good poin. Yes

**Review by tenphi — APPROVED** [review]

File: src/components/pickers/Menu/Menu.stories.tsx (hunk 2 above)
Comment (nikolaykost) [reply src/components/pickers/Menu/Menu.stories.tsx:95]: Done. Added play test

**Review by MrFlashAccount — APPROVED** [review]
LGTM 🫡

**Pull request merged**